Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -3161,37 +3116,155 @@ enum class WebCore::ApplePayButtonStyle : uint8_t {
     Black,
 }
 
-[RefCounted, AdditionalEncoder=StreamConnectionEncoder] class WebCore::ApplePayButtonPart {
+[AdditionalEncoder=StreamConnectionEncoder] class WebCore::ApplePayButtonAppearance {
     WebCore::ApplePayButtonType buttonType();
     WebCore::ApplePayButtonStyle buttonStyle();
-    String locale();
+    WTF::String locale();
+};
+#endif
+
+using WebCore::ControlAppearance = std::variant<
```

Comment:
Why do we have to repeat this definition here? I was able to encode the DisplayListItem which is also variant without repeating its definition. See the changes in DisplayListArgumentCoders.serialization.in in https://github.com/WebKit/WebKit/pull/16688.

Reply:
```
    for using_statement in using_statements:
        if using_statement.condition is not None:
            result.append('#if ' + using_statement.condition)
        result.append('        { "' + using_statement.name + '"_s, {')
        result.append('            { "' + using_statement.alias + '"_s, "alias"_s }')
        result.append('        } },')
        if using_statement.condition is not None:
            result.append('#endif')